CompatibilityReport/CompatibilityReport.csproj

```diff
@@ -132,6 +132,9 @@
       <LastGenOutput>HtmlReportTemplate.cs</LastGenOutput>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
```

Comment:
what is that?

Reply:
> what is that?

Good question, no clue where this come from...

Reply:
@krzychu124 should i remove that?